modules/pool/lib/pool-gql-loader.service.ts

```diff
@@ -970,6 +980,63 @@ export class PoolGqlLoaderService {
         };
     }
 
+    private buildAprItems(pool: PrismaPoolMinimal): GqlPoolAprItem[] {
+        const aprItems: GqlPoolAprItem[] = [];
+
+        for (const aprItem of pool.aprItems) {
+            if (aprItem.apr === 0 || (aprItem.range && aprItem.range.max === 0)) {
+                continue;
+            }
+
+            let type: GqlPoolAprItemType = 'STAKING';
+            switch (aprItem.type) {
+                case PrismaPoolAprType.NATIVE_REWARD:
+                case PrismaPoolAprType.THIRD_PARTY_REWARD:
+                    type = 'STAKING';
+                    break;
+                case PrismaPoolAprType.IB_YIELD:
+                    type = 'IB_YIELD';
+                    break;
+                case PrismaPoolAprType.LOCKING:
+                    type = 'LOCKING';
+                    break;
+                case PrismaPoolAprType.SWAP_FEE:
+                    type = 'SWAP_FEE';
+                    break;
+                case PrismaPoolAprType.VOTING:
+                    type = 'VOTING';
+                    break;
+                case null:
+                    type = 'NESTED';
+                    break;
+            }
+
+            if (aprItem.range) {
+                aprItems.push({
+                    id: aprItem.id,
+                    title: aprItem.title,
+                    apr: aprItem.range.min.toString(),
```

Comment:
can we store them as numbers? it will be easier to do any sql on them

Reply:
they are stored as numbers. You mean also return it as a float in the query?